CHANGELOG.md

```diff
@@ -1,5 +1,145 @@
 # Changelog
 
+## 0.14.0 (2024-08-01)
+
+This release updates to [Rustls 0.23.12][] and changes the rustls-ffi API to allow
+choosing a cryptography provider to use with Rustls. 
+
+The default provider has been changed to match the Rustls default,
+[`aws-lc-rs`][]. Users that wish to continue using `*ring*` as the provider may
+opt-in. See the `README` for more detail on supported platforms and build
+requirements.
+
+[Rustls 0.23.12]: https://github.com/rustls/rustls/releases/tag/v%2F0.23.12
+[`aws-lc-rs`]: https://github.com/aws/aws-lc-rs
+
+### Added
+
+* A new `rustls_crypto_provider` type has been added to represent
+  `rustls::CryptoProvider` instances.
+  * The current process-siwde default crypto provider (if any) can be retrieved
+    with `rustls_crypto_provider_default()`.
+  * If rustls-ffi was built with `aws-lc-rs`, (`DEFINE_AWS_LC_RS` is true), then
+    `rustls_aws_lc_rs_crypto_provider()` can be used to retrive the `aws-lc-rs`
+    provider.
+  * If rustls-ffi was built with `*ring*`, (`DEFINE_RING` is true), then
```

Comment:
```suggestion
  * If rustls-ffi was built with `ring`, (`DEFINE_RING` is true), then
```

I would avoid the stylised name in code fonts that suggest literal uses.

Reply:
Fixed throughout (though I left a historic mention in older release notes).